sdk/src/VeraxSdk.ts

```diff
@@ -189,10 +189,12 @@ export class VeraxSdk {
       });
     }
 
-    this.attestation = new AttestationDataMapper(conf, this.web3Client, this, this.walletClient);
-    this.schema = new SchemaDataMapper(conf, this.web3Client, this, this.walletClient);
-    this.module = new ModuleDataMapper(conf, this.web3Client, this, this.walletClient);
-    this.portal = new PortalDataMapper(conf, this.web3Client, this, this.walletClient);
-    this.utils = new UtilsDataMapper(conf, this.web3Client, this, this.walletClient);
+    this.schema = new SchemaDataMapper(conf, this.web3Client, this.walletClient);
+    const findOneSchemaById = this.schema.findOneById.bind(this.schema);
```

Comment:
Not sure if it's correct to pass `this.schema` as the function parameter to `findOneById.bind()`, iiuc, the function parameter should be an string given that `async findOneById(id: string)`, not sure why here it was passed with the `SchemaDataMapper` object, also `bind` would not do any type check, so better avoid to use it if possible, see [here](https://hamednourhani.gitbooks.io/typescript-book/content/docs/tips/bind.html)